Mac Docker Config Problem on Step 8

On Step 8, I ran `make build` but received the following error. Any thoughts @aileenzeng?

```
...
Successfully tagged makeability:latest
docker run -p 8000:8000 -ti -v database:/code/db -v $(pwd)/media:/code/media $(pwd)/website:/code/website makeability
docker: invalid reference format: repository name must be lowercase.
See 'docker run --help'.
make: *** [run] Error 125
```

--- a/Makefile
+++ b/Makefile
@@ -13,7 +13,7 @@ build:
 	docker build . -t $(IMAGE_NAME)
 
 run: build
-	docker run -p 8000:8000 -ti -v database:/code/db -v $$(pwd)/media:/code/media $$(pwd)/website:/code/website $(IMAGE_NAME)
+	docker run -p 8000:8000 -ti -v database:/code/db -v $$(pwd)/media:/code/media -v $$(pwd)/website:/code/website $(IMAGE_NAME)
 
 gitpull:
 	git pull origin master

